HISTORY.md

```diff
@@ -1,5 +1,12 @@
 # Version History
 
+## 1.1.0 / 2022-02-28
+
+- Updated dependencies
+- Migrated build agent
+- Update analyses
```

Comment:
Super picky comment but these are in different tenses

Reply:
Tense fixed!